sql/catalyst/benchmarks/GenericArrayDataBenchmark-jdk11-results.txt

```diff
@@ -0,0 +1,10 @@
+OpenJDK 64-Bit Server VM 11.0.5+10 on Mac OS X 10.14.6
+Intel(R) Core(TM) i5-8210Y CPU @ 1.60GHz
+constructor:                              Best Time(ms)   Avg Time(ms)   Stdev(ms)    Rate(M/s)   Per Row(ns)   Relative
+------------------------------------------------------------------------------------------------------------------------
+arrayOfAny                                            6              7           1       1770.9           0.6       1.0X
+arrayOfAnyAsObject                                    6              7           2       1709.3           0.6       1.0X
+arrayOfAnyAsSeq                                       5              6           2       2195.5           0.5       1.2X
+arrayOfInt                                          452            469          13         22.1          45.2       0.0X
+arrayOfIntAsObject                                  678            690          11         14.7          67.8       0.0X
+
```

Comment:
Here are the JDK11 benchmark results before this patch's changes:

```
[info] OpenJDK 64-Bit Server VM 11.0.5+10 on Mac OS X 10.14.6
[info] Intel(R) Core(TM) i5-8210Y CPU @ 1.60GHz
[info] constructor:                              Best Time(ms)   Avg Time(ms)   Stdev(ms)    Rate(M/s)   Per Row(ns)   Relative
[info] ------------------------------------------------------------------------------------------------------------------------
[info] arrayOfAny                                            6              6           1       1776.5           0.6       1.0X
[info] arrayOfAnyAsObject                                  374            386           9         26.7          37.4       0.0X
[info] arrayOfAnyAsSeq                                       5              5           0       2211.1           0.5       1.2X
[info] arrayOfInt                                          472            494          29         21.2          47.2       0.0X
[info] arrayOfIntAsObject                                  798            799           1         12.5          79.8       0.0X
```

Reply:
Thank you so much!

Reply:
A pleasant discovery is that this patch has an even larger positive impact on JDK11: on JDK11, this PR's changes seem to more-or-less completely eliminate the performance gap between the `this(Any)` and `this(Array[Any])` constructors 🎉

Reply:
oh, the result looks nice.